### ISSUE TITLE
bring Tenant and Region Switch as part of the shell-bar

### ISSUE THREAD

**Issue by droshev** [SAP/fundamental-ngx#669]
#### Is this a bug, enhancement, or feature request?
enhancement

#### Briefly describe your proposal.
Once fiori fundamentals implement https://github.com/SAP/fundamental-styles/issues/117 then we should be able to provide the tenant and region switcher

**Comment by droshev** [comment]
blocked by  SAP/fundamental#1368

**Comment by InnaAtanasova** [comment]
Out of scope
